test/integration/service_manual_guide_test.rb

```diff
@@ -16,4 +16,48 @@ class ServiceManualGuideTest < ActionDispatch::IntegrationTest
       refute page.has_content?('Published by')
     end
   end
+
+  test "service manual guide shows latest change history" do
+    setup_and_visit_content_item('with_change_history')
+    within ".change-history" do
+      within ".change-history-published-by" do
+        assert page.has_content? "Agile delivery community"
+      end
+
+      notes = all(".change-history-note").map(&:text)
+      assert_includes notes, "This is a change"
+
+      reasons = all(".change-history-reason").map(&:text)
+      assert_includes reasons, "This is our latest change"
+
+      timestamps = all(".change-history-public-timestamp").map(&:text)
+      assert_includes timestamps, "09 October 2017"
+    end
+  end
+
+  test "service manual guide hides older change history" do
+    setup_and_visit_content_item('with_change_history')
+
+    module_container = find(".change-history")
+    assert_equal "toggle", module_container["data-module"]
+    assert_equal "toggle", module_container["data-module"]
+
+    hidden_container = find(".change-history #change-history-hidden")
+    assert_equal "js-hidden", hidden_container["class"]
+
+    expand_link = find(".change-history a", text: "+ full page history")
+    assert_equal "false", expand_link["data-expanded"]
+    assert_equal "change-history-hidden", expand_link["data-controls"]
+
+    within ".change-history #change-history-hidden" do
+      notes = all(".change-history-note").map(&:text)
+      assert_includes notes, "This is another change"
+
+      reasons = all(".change-history-reason").map(&:text)
+      assert_includes reasons, "This is why we made this change and it has a second line of text"
+
+      timestamps = all(".change-history-public-timestamp").map(&:text)
+      assert_includes timestamps, "09 January 2016"
+    end
+  end
```

Comment:
Personally I'm unsure of this business of checking that some elements have certain classes rather than checking the classes are doing what we want them to do. eg. use a real browser driver such an phantomjs or selenium, rather than rack-test, and check something isn't visible rather than check `js-hidden` is present.

Maybe these things are tested elsewhere, we can trust it, and testing like this is sensible or perhaps we just don't have to bother testing it here?

Maybe @dsingleton or @fofr could guide us here?